src/sections/image-with-text.liquid

```diff
@@ -0,0 +1,110 @@
+{%- comment -%}
+  This is a required section for the Shopify Theme Store.
+  It is available when you add "Image With Text" section on the Theme Editor.
+
+  Theme Store required settings
+  - Image
+  - Image alignment: to align your image left or right
+
+  Theme Store optional settings
+  - Heading: title of the block
+  - Text: description of the block
+  - Button label: Label associated with the button
+  - Button link: Link of the button
+{%- endcomment -%}
+
+<div class="image-with-text">
+  {% capture image_layout %}
+    <div class="image-with-text__image">
+      {% if section.settings.image != blank %}
+        {% include 'responsive-image' with
+          image: section.settings.image,
+          max_width: 545,
+          max_height: 545
+        %}
+      {% else %}
+        {{ 'image' | placeholder_svg_tag: 'placeholder-svg' }}
+      {% endif %}
+    </div>
+  {% endcapture %}
+
+  {% if section.settings.layout == 'left' %}
+    {{ image_layout }}
+  {% endif %}
+
+  <div>
+    {% if section.settings.title != blank %}
+      <h2>{{ section.settings.title | escape }}</h2>
+    {% endif %}
+    {% if section.settings.text != blank %}
+      <div>{{ section.settings.text }}</div>
```

Comment:
`{{ section.settings.text | escape }}`

Reply:
Actually @chrisberthe , I have to revert back because that field is a "Rich Text Editor" and we see the HTML markup as a string in the output... 😢

Reply:
Ohhh right, woopsies.